examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -767,6 +783,280 @@
     "`ConditionalDimension` can be used as an alternative to save snaps to disk as illustrated in [08_snapshotting](https://github.com/devitocodes/devito/blob/master/examples/seismic/tutorials/08_snapshotting.ipynb \"08_snapshotting\"). The example subsamples the `TimeDimension`, capturing snaps of the our wavefield at equally spaced snaps. "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example A - Count nonzero elements\n",
+    "\n",
+    "Here we present an of using `ConditionalDimension`. In the following example, an `Operator` counts the nonzero elements of an array."
```

Comment:
"an of using" -- to be fixed @georgebisbas

Reply:
Fixed